pycrostates/viz/tests/test_cluster_centers.py

```diff
@@ -1,14 +1,20 @@
+import logging
+
 import numpy as np
 import pytest
 from matplotlib import pyplot as plt
 from mne import create_info
 
 from pycrostates.io import ChInfo
+from pycrostates.utils._logs import logger
 from pycrostates.viz import plot_cluster_centers
 
+logger.propagate = True
+
 
-def test_plot_cluster_centers():
+def test_plot_cluster_centers(caplog):
     """Test topographic plots for cluster_centers."""
+    caplog.set_level(logging.WARNING)
```

Comment:
I had no idea you could do that! Good to know, usually I set the logger level before testing ^^'

Reply:
I just copied it from another test file in the library 👀